SU-SWS/decanter
# Nav placeholders, color map, styles, twig files

# READY FOR REVIEW

# Summary
- Separating the styles related changes in the DEC-71 PR so they could be merged into master earlier.
- Also pulled in new main-nav.twig and menu-loop.twig files from #443 

# Needed By (Date)
- Soon

# Urgency
- N/A

# Steps to Test

1. Pull this branch and compile styleguide
1. Check that the Main Nav component and variants have same styles/colors as the one on the live site.
1. (for @JBCSU ). Have Redwood use this branch of Decanter, and see if the main nav still works as is with the new twig file and styles.

# Affected Projects or Products
- Decanter
- Redwood WordPress theme

# Associated Issues and/or People
- #443
